Sources/StreamChatUI/ChatChannelList/Search/ChatMessageSearchVC.swift

```diff
@@ -105,6 +109,10 @@ open class ChatMessageSearchVC: ChatChannelListSearchVC, ChatMessageSearchContro
 
     // MARK: - ChatMessageSearchControllerDelegate
 
+    override open func controller(_ controller: DataController, didChangeState state: DataController.State) {
+        reloadMessages()
+    }
```

Comment:
It is not super great because this class is a subclass of a channel list VC but here we are dealing with messages instead. Therefore, we need to override superclass implementation which does not know about triggering reloading messages.

Reply:
How is this not a problem on the Channel Search? 🤔